### ISSUE TITLE
Duplicate log threads while resizing the Bash window

### ISSUE THREAD

**Issue by Gideonamani** [git-for-windows/git#841]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen)
  or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue
  matching what I'm seeing
### Setup

 Am using the 32-bit version

```
$ git --version
git version 2.8.1.windows.1
```

I am running Windows 7 32_bit

I think I chose the default options during installation
### Details
- Which terminal/shell was I running Git from?

Bash
- What commands I ran to trigger this issue?

```
 $ git log --graph --oneline master
```
- What did I expect to occur after running these commands?

The log of the list of all commits
- What actually happened instead?

All the commits did appear, but the problem is when I resize my Bash window the list of commits gets duplicated. 
That is the extract of the list of commits that gets displayed 

```
.
.
.
* 292faae added bullets
* daebb75 added bullets
* fc72f92 Add README
* 505f221 initial add
~
(END)
```

and when I resize the window, I again get a thread of commits which get displayed again below another. So when I now scroll up to look at the log I see multiple (END)s and duplicate threads corresponding to the number of times I resized my window. 

```
.
.
.
* 292faae added bullets
* daebb75 added bullets
* fc72f92 Add README
* 505f221 initial add
~

(END)

.
.
.
* 292faae added bullets
* daebb75 added bullets
* fc72f92 Add README
* 505f221 initial add
~
(END)
```

Thank you :) :100:

**Comment by dscho** [comment]
Is this still happening?

**Comment by dscho** [comment]
> Is this still happening?

To be precise: I tried on multiple occasions, spending hours and hours, to reproduce this from the given description. I never succeeded in reproducing the problem.

That leaves me with only option to continue on this bug: to rely on the feedback of the original poster to try to work something out.

As there is no feedback, and as I already spent more time on this ticket than I reasonably can (there are more tickets open in this here bug tracker, after all, that affect many more users than just a single one), I am actually left with only one choice: to close the ticket.
